### ISSUE TITLE
error when installing

### ISSUE THREAD

**Issue by eturino** [seawatts/ember-simple-auth-auth0#1]
```
$ ember i ember-simple-auth-auth0                                                                                                                                                                                                                                       
Installed packages for tooling via npm.
installing ember-simple-auth-auth0
The `ember generate <entity-name>` command requires an entity name to be specified. For more details, use `ember help`.
```

**Comment by eturino** [comment]
tried it in a fresh new ember 2.7 app, same result

**Comment by eturino** [comment]
apparently it's solved by adding this to the blueprint:

```
  normalizeEntityName: function() {
  },
```

since this blueprint does not need an argument. I'll have a look to see if there is a cleaner way (I haven't built any addon yet) and I'll make a PR later.

**Comment by seawatts** [comment]
fixed in #2 and released https://github.com/seawatts/ember-simple-auth-auth0/releases/tag/v0.1.2
